applications/portal/backend/api/migrations/0006_remove_antibody_target_species_raw.py

```diff
@@ -0,0 +1,65 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0005_alter_antibody_lastedit_time_and_more'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+            DROP MATERIALIZED VIEW IF EXISTS antibody_search;
+            CREATE MATERIALIZED VIEW antibody_search AS 
+            SELECT api_antibody.ix, 
+            (
+                setweight(to_tsvector('english'::regconfig, 
+                    COALESCE(ab_name, ''::text) || ' ' ||
+                    COALESCE(clone_id, ''::text)
+                ), 'A'::"char") ||
+                
+                setweight(to_tsvector('english'::regconfig, 
+                    COALESCE(api_vendor.vendor, ''::text) || ' ' ||
+                    COALESCE(api_specie.name, ''::text) || ' ' || 
+                    COALESCE(STRING_AGG(related_species.name, ' '), '') || ' ' ||
```

Comment:
@filippomc please be aware of this change. I don't really know how to properly test it